src/stk/molecular/atoms/atom.py

```diff
@@ -15,103 +17,107 @@ class Atom:
     Atoms of a particular element can be made with this
     class or with the subclass representing that element.
 
-    Examples
-    --------
-    *Initialization.*
+    Examples:
+
+        *Initialization.*
 
-    Initialization of an :class:`.Atom` can happen in one of two ways.
-    The atom can be initialized through the :class:`.Atom` class or
-    through the class representing the element.
+        Initialization of an :class:`.Atom` can happen in one of two
+        ways. The atom can be initialized through the :class:`.Atom`
+        class or through the class representing the element.
 
-    .. testcode:: initialization
+        .. testcode:: initialization
 
-        import stk
+            import stk
 
-        # h0 is an instance of the H class.
-        h0 = stk.Atom(id=0, atomic_number=1)
+            # h0 is an instance of the H class.
+            h0 = stk.Atom(id=0, atomic_number=1)
 
-        # h1 is also an instance of the H class.
-        h1 = stk.H(id=1)
+            # h1 is also an instance of the H class.
+            h1 = stk.H(id=1)
 
-    When the class corresponding to the element is used directly, the
-    ``atomic_number`` is not provided. Here are a few more examples.
+        When the class corresponding to the element is used directly,
+        the ``atomic_number`` is not provided. Here are a few more
+        examples.
 
-    .. testcode:: initialization
+        .. testcode:: initialization
 
-        # Both he0 and he1 are instances of the He class.
-        he0 = stk.Atom(id=2, atomic_number=2)
-        he1 = stk.He(id=3)
+            # Both he0 and he1 are instances of the He class.
+            he0 = stk.Atom(id=2, atomic_number=2)
+            he1 = stk.He(id=3)
 
-        # Both c0 and c1 are instances of the
-        # C class.
-        c0 = stk.Atom(id=4, atomic_number=6)
-        c1 = stk.C(id=5)
+            # Both c0 and c1 are instances of the
+            # C class.
+            c0 = stk.Atom(id=4, atomic_number=6)
+            c1 = stk.C(id=5)
 
     """
 
     # Maps each atomic number (int) to the relevant Atom subclass.
-    _elements = {}
+    _elements: Dict[int, Type['Atom']] = {}
+    _atomic_number: ClassVar[int]
 
-    def __init_subclass__(cls, **kwargs):
+    def __init_subclass__(cls: Type['Atom'], **kwargs) -> None:
```

Comment:
Why not cls: Atom ?

Reply:
Because `cls; Atom` is saying `cls` is an in instance of `Atom` while `Type['Atom']` is saying `cls` is a subclass of `Atom`

Reply:
Did not know. Thank you!